#!/bin/sh -e
## 01_texinfo-usr-local.dpatch by James Troup <james@nocrew.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix /usr/local references in Texinfo manual.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/james/debian/packages/gdbm173/gdbm173-1.7.3/gdbm.texinfo gdbm173-1.7.3/gdbm.texinfo
--- /home/james/debian/packages/gdbm173/gdbm173-1.7.3/gdbm.texinfo	1994-05-19 23:14:11.000000000 +0100
+++ gdbm173-1.7.3/gdbm.texinfo	2003-05-05 17:48:36.000000000 +0100
@@ -209,7 +209,7 @@
 int gdbm_setopt(dbf, option, value, size)
 @end example
 
-The @code{gdbm.h} include file is often in the @file{/usr/local/gnu/include}
+The @code{gdbm.h} include file is often in the @file{/usr/include}
 directory. (The actual location of @code{gdbm.h} depends on your local
 installation of @code{gdbm}.)
 
@@ -665,7 +665,7 @@
 and want to use @code{gdbm} files, execute the following @code{cc} command:
 
 @example
-cc ... -L /usr/local/lib -lgdbm
+cc ... -lgdbm
 @end example
 
 @node Conversion, Bugs, Compatibility, Top
